kubric/traits.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The Kubric Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import traitlets as tl
+
+from kubric.color import Color
```

Comment:
Yes, import paths are something that we should fix for sure!
IMHO many things should be importable from the `kubric` package directly.

(Good to know! I did not realize that the style guide prevents these imports. I will have to fix quite a few then....)

BTW: While it is nice and explicit that the `Color` class has separate constructors, I find it quite annoying to write `Color.from_name("white")` etc.. Maybe it would be acceptable to have a generic constructor that infers the format of the arguments (`Color("white")`, `Color(0xffffff)`, `Color('#fffff')`, ...)